elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -1329,12 +1329,12 @@ public void patchExtBadValue() throws IOException {
             .asString());
 
         JsonNode errors = result.get("errors");
-        Assert.assertNotNull(errors);
-        Assert.assertEquals(errors.size(), 1);
+        assertNotNull(errors);
+        assertEquals(errors.size(), 1);
 
         String error = errors.get(0).asText();
-        String expected = "TransactionException: Duplicate entry 'duplicate' for key 'name'";
-        Assert.assertTrue(error.equals(expected), "Error does not equal with '" + expected + "'");
+        String expected = "TransactionException: Unique index or primary key violation:";
+        assertTrue(error.startsWith(expected), "Error does not start with '" + expected + "' but found " + error);
```

Comment:
minor question: why the change here from `equals` to `startsWith`?

Reply:
The actual message is 
```
TransactionException: Unique index or primary key violation: "CONSTRAINT_INDEX_5 ON PUBLIC.CHILDENTITY(NAME) VALUES (CAST('duplicate' AS VARCHAR_IGNORECASE), 10)"
```

Reply:
Additional change from #739 because Hibernate 3 and 5 throw different error strings.